src/convnets/builders/mbconv.jl

```diff
@@ -12,13 +17,14 @@ function dwsepconv_builder(block_configs, inplanes::Integer, stage_idx::Integer,
                                stride, pad = SamePad(), norm_layer, kwargs...)...)
         return (block,)
     end
-    return get_layers, nrepeats
+    return get_layers, ceil(Int, nrepeats * depth_mult)
 end
+_get_builder(::typeof(dwsep_conv_norm)) = dwsepconv_builder
```

Comment:
Perhaps this is what your comment above is alluding to, but as-is it seems like `_get_builder(::typeof(f))` and `f_builder` could be merged into one function.

Reply:
They _could_, but then the config dict has to specify the builder function and not the block function

Reply:
Not sure I follow? What I meant was that the function currently called `dwsepconv_builder(args...)` could just be `_get_builder(::typeof(dwsep_conv_norm), args...)`. AFAICT this should not change what is stored in the config dict because it simply eliminates one function dispatch from the call chain.

Reply:
Ah, I didn't think about eliminating the explicit builder altogether and just making it dispatch. That makes an awful lot of sense. I still keep forgetting multiple dispatch exists sometimes

Reply:
Well, you _are_ using it already for `_get_builder`! This merging of functions just happens to be something that came out of the wash while refactoring the `mbconv_stage_builder`.

Reply:
Yep, this is done now - perhaps the only nitpick is perhaps that to allow this kind of structure there is a redundant `stochastic_depth_prob` argument now being passed in to the depthwise separable convolution builder. But it's a small price to pay for uniform code